src/modules/flow/twitter/twitter.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "http://solettaproject.github.io/soletta/schemas/node-type-genspec.schema",
+  "name": "twitter",
+  "meta": {
+    "author": "Intel Corporation",
+    "license": "BSD-3-Clause",
+    "version": "1"
+  },
+  "types": [
+    {
+      "name": "twitter/client",
+      "category": "output/network",
+      "description": "A twitter's node client",
+      "methods": {
+        "close": "twitter_close",
+        "open": "twitter_open"
+      },
+      "options": {
+	"members": [
+	  {
+	    "data_type": "string",
+	    "description": "The consumer key (application's token)",
+	    "name": "consumer_key"
+	  },
+	  {
+	    "data_type": "string",
+	    "description": "The consumer secret key (application's secret token)",
+	    "name": "consumer_secret"
+	  }
+	],
+	"version": 1
+      },
+      "in_ports": [
+        {
+          "data_type": "string",
+          "description": "The token returned on twitter's oauth process",
+          "methods": {
+            "process": "token_process"
+          },
+          "name": "TOKEN"
+        },
+	{
+          "data_type": "string",
+          "description": "Post the string that comes into twitter",
+          "methods": {
+            "process": "post_process"
+          },
+          "name": "TWEET"
```

Comment:
we'll never cover all APIs by adding ports. I believe a more extensible approach is to create one node per supported function, like `twitter/tweet` and `twitter/get-timeline`. This also allow us to offer some relevant input or output ports in cases where these are easier to do.

Like `twitter/tweet` could take multiple ports, one to specify a location (that we could get from gps), one to get the text and another `POST` to do the action.